dali/server/daldap.cpp

```diff
@@ -144,12 +144,21 @@ class CDaliLdapConnection: public CInterface, implements IDaliLdapConnection
             Owned<ISecUser> user = ldapsecurity->createUser(username);
             if (user) {
                 user->credentials().setPassword(password);
-                if (filescope)
-                    perm=ldapsecurity->authorizeFileScope(*user, obj);
-                else if (wuscope)
-                    perm=ldapsecurity->authorizeWorkunitScope(*user, obj);
-                if (perm==-1)
-                    perm = 0;
+                bool superUser;
+                if (!ldapsecurity->authenticateUser(*user, superUser))
```

Comment:
Calculating superUser (which is then ignored) looks like it requires an ldap call.  Should we try and avoid this?